_data/speakers.yml

```diff
@@ -248,17 +256,18 @@
   image: helleckes_07.jpg
   thumbnail: helleckes_07.jpg
   alt: Laura
-  topic:
-  proposal_summary:
-  discourse_link:
+  topic: calibr8: Going beyond linear ranges with non-linear calibration curves and multilevel modeling
+  proposal_summary: You just coded up a beautiful model and dummy prediction looks great. Now comes the data, but wait: the units don't match! And to make matters worse, the correlation between model variable and measurement readout is non-linear and heteroscedastic! Sounds familiar? non-linear calibration to the rescue! With `calibr8`, we present a statistical framework and corresponding open source Python package that solves non-linear calibration and likelihood functions for modeling. From a laboratory automation and systems biology perspective, the advantage of non-linear calibration with `calibr8` is two-fold: For lab scientists doing (high-throughput) experiments, `calibr8` facilitates more intuitive uncertainty quantification and makes every-day data analysis more robust, automatable and Bayesian. From a modeling perspective, non-linear _error models_ are essential components of realistic Bayesian process models, and are key to accurately describe the nastiest step of the data-generating process. In this talk, we will take you step-by-step through the data-generating process of an automated bioassay and demonstrate how its non-linearities are modeled with `calibr8`. We will show how `calibr8` can make your life easier - and of course more Bayesian - even if you don't always go all the way to a process model. Finally, we will show how non-linear error models and multi-level modeling with `PyMC3` enable you to get more information out of heterogeneous regression analyses. Join us for the talk and discussion to learn about building Bayesian models for bioassays and dive into the fascinatingly frightening world of non-linear measurement errors!
+  discourse_link: 
   youtube_link:
   description: A biotechnologist by training, Laura transitioned to Data Science in the past years and is now a Bayesian enthusiast.
     In her Master’s thesis, she _actually_ collected the data Michael was using for his fancy Bayesian models.
     During her wet lab experience, Laura gained valuable knowledge on microorganisms and biological processes that she is now applying to implement mechanistic process models.
     Her experimental work also gave her the motivation to focus on lab automation for bioprocess development in her PhD at Forschungszentrum Jülich.
   presentation_type: Talk
-  time_zone:
+  time_zone: Asia/Africa/Europe
   track: Beginner
+  keywork: Laukes
```

Comment:
The keyword needs to be one per talk, otherwise the grouping won't work. Here however, Laura has `Laukes` and below Michael Osthege has `Micege`. They are giving a talk between the two. In order to be able to group them, they should both have the same keyword, maybe `calibr8` or anything else, but it must be one per talk, not one per speaker. Sorry to be so nitpicky, but otherwise the grouping won't work

Reply:
Did this